test/automation/src/sql/managePackagesDialog.ts

```diff
@@ -60,4 +61,41 @@ export class ManagePackagesDialog extends Dialog {
 
 		return packageVersion;
 	}
+
+	async removePackage(packageName: string, clickOnTab: boolean = false): Promise<void> {
+		if (clickOnTab) {
```

Comment:
When the dialog is opened we're already on the Installed Packages page, so trying to click on the tab to select the page fails. This option allows the page to be clicked on if we're interleaving install and uninstall operations in an already open dialog.

Reply:
Add comment explaining this in the code